gapis/gfxapi/templates/api.go.tmpl

```diff
@@ -127,6 +127,11 @@
     {{$to := Macro "Go.Type" $.To}}
 
     type {{$.Name}} {{$to}}
+    {{if (GetAnnotation $ "dispatchHandle")}}
```

Comment:
But I remember so far we don't have types defined with int, uint, and char. Types in Vulkan are defined as uint32_t, int32_t according to Spec.